### PR TITLE
 Create workflow for auto assignment of issues and for stale issues

### PR DESCRIPTION
**Pull request by sachinprasadhs**

### CODE REVIEW

File: .github/workflows/stale-issue-pr.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Close inactive issues
```
Comment (mattdangerw) [inline .github/workflows/stale-issue-pr.yml:1]: Will this affect existing issues? We probably have some old stale ones.

File: .github/workflows/stale-issue-pr.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          only-labels: "stat:awaiting response from contributor"
+          stale-issue-message: > 
+            This issue is stale because it has been open for 14 days with no activity.
+            It will be closed if no further activity occurs. Thank you.
+          # List of labels to remove when issues/PRs unstale. 
+          labels-to-remove-when-unstale: "stat:awaiting response from contributor"
+          close-issue-message: >
+            This issue was closed because it has been inactive for 28 days.
+            Please reopen if you'd like to work on this further.
+          days-before-pr-stale: 14
```
Comment (mattdangerw) [inline .github/workflows/stale-issue-pr.yml:29]: Does this mean all prs with no activity will close after 14 days? Or just ones annotated with awaiting response?

**Review by mattdangerw — COMMENTED** [review]
Code looks good. Just have some questions as to what this will do.

File: .github/workflows/stale-issue-pr.yml (hunk 1 above)
Comment (sachinprasadhs) [reply .github/workflows/stale-issue-pr.yml:1]: This applies only when the label "stat:awaiting response from contributor" is applied

File: .github/workflows/stale-issue-pr.yml (hunk 2 above)
Comment (sachinprasadhs) [reply .github/workflows/stale-issue-pr.yml:29]: This applies only when the label "stat:awaiting response from contributor" is applied. For the important PRs we can avoid applying the label.

Comment (mattdangerw) [reply .github/workflows/stale-issue-pr.yml:29]: Got it! Let's try it out!

**Pull request merged**